freepl_main/settings.py.example

```diff
@@ -12,6 +12,7 @@ https://docs.djangoproject.com/en/1.8/ref/settings/
 
 # Build paths inside the project like this: os.path.join(BASE_DIR, ...)
 import os
+from decouple import config
```

Comment:
Add `python-decouple` entry in `requirements.txt`

Reply:
done